src/utils/changed.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Returns true if any of the given props are different between given objects.
+ * Only shallow changes are detected.
+ */
+export default (props, objectA, objectB) => {
+  return (
+    objectA && objectB && props.some(prop => objectA[prop] !== objectB[prop])
+  );
+};
```

Comment:
I would've been more inclined to put this in the main `utils/index.js` file but it honestly doesn't matter